package.json

```diff
@@ -17,7 +17,10 @@
   "scripts": {
     "start": "grunt",
     "test": "grunt test",
-    "postinstall": "bower install --config.interactive=false"
+    "postinstall": [
+      "bower install --config.interactive=false",
+      "./node_modules/protractor/bin/webdriver-manager update"
```

Comment:
Can this be added as a Grunt task that is coupled with the protractor task? I'm concerned that not all users will want to deal with protractor & the e2e tests at the time of package install.